tastytrade/instruments.py

```diff
@@ -1,6 +1,6 @@
 from dataclasses import dataclass
 from datetime import date, datetime
-from typing import Any, Optional
+from typing import Any, Optional, List
```

Comment:
I've been using lowercase `list` for typing in the rest of the module, let's keep that the same.

Reply:
Done